internal/sysfs/osfile.go

```diff
@@ -98,9 +98,32 @@ func (f *osFile) reopen() (errno experimentalsys.Errno) {
 	// Clear any create flag, as we are re-opening, not re-creating.
 	f.flag &= ^experimentalsys.O_CREAT
 
+	var (
+		offset int64
+		err    error
+	)
```

Comment:
I don't think we normally use this style for inline declarations; besides err is only used in the nested scope, so maybe you can declare `var offset int64` here...